README.adoc

```diff
@@ -144,58 +142,44 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=InventoryManager]
 ----
 
-Notice the metrics annotations added to this class:
-
-The `@Timed` annotation applies to the `get()` method to track how frequently the method is
+Apply the `@Timed` annotation to the `get()` method to track how frequently the method is
 invoked and also how long it takes for the invocation to complete. This annotation has these
 metadata fields:
 
 |===
-|`name` | Optional and sets the name of the metric
-|`description` | Optional and describes the purpose of the metric annotation
+|`name` | Optional; Use this to name of the metric
+|`description` | Optional; Use this to describe the purpose of the metric
```

Comment:
Pretty much the same changes as the line above:
"Optional. Use this annotation to describe the purpose of the metric."